src/components/Book/BookItem/index.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default function Book({ book }) {
+  return (
+    <div>
+      <h3>{book.title}</h3>
+      <p>{book.author}</p>
+    </div>
```

Comment:
- [ ] This component should have a button to delete a book from the Books list, kindly make sure to implement this so the books can be deleted